src/main/java/io/github/genomicdatainfrastructure/discovery/services/PackageShowMapper.java

```diff
@@ -0,0 +1,105 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package io.github.genomicdatainfrastructure.discovery.services;
+
+import java.util.List;
+import java.util.Objects;
+
+import io.github.genomicdatainfrastructure.discovery.model.RetrievedDataset;
+import io.github.genomicdatainfrastructure.discovery.model.RetrievedDistribution;
+import io.github.genomicdatainfrastructure.discovery.model.ValueLabel;
+import io.github.genomicdatainfrastructure.discovery.remote.ckan.model.CkanOrganization;
+import io.github.genomicdatainfrastructure.discovery.remote.ckan.model.CkanPackage;
+import io.github.genomicdatainfrastructure.discovery.remote.ckan.model.CkanResource;
+
+import static java.util.Optional.ofNullable;
+import static java.util.function.Predicate.not;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class PackageShowMapper {
+
+    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern(
+            "yyyy-MM-dd'T'HH:mm:ss.SSSSSS"
+    );
+
+    private PackageShowMapper() {
```

Comment:
what's the thing with these empty constructors, we need to have them?

Reply:
These are utility classes. If they have only static methods, the constructor have to be private and a comment must be provided. But this is just a convention checked by Sonar.

Reply:
Particularly to this convention, I think it does make sense to make the constructor private to ensure this class contains only behavior and must not be instanciated - As it's an utility class. An empty private constructor if just the way to do it in Vanila Java. What we can do is to find other ways to do that with lombok. 

I mentioned Sonar just because it usually checks this convension, but I agree with you we should not blindly follow it. 

Please, few free to review our current quality gate and suggest changes: https://sonarcloud.io/organizations/genomicdatainfrastructure/quality_profiles/show?name=Sonar+way&language=java